gramex/apps/nlg/grammar.py

```diff
@@ -3,15 +3,15 @@
 from spacy.lemmatizer import Lemmatizer
```

Comment:
Please lazy-load spacy. To test it, uninstall spacy and run the guide. It should still work.